pkg/plugins/autodiscovery/githubaction/main_test.go

```diff
@@ -96,6 +97,136 @@ conditions:
       url: "https://github.com/actions/checkout.git"
       password: 'xxx'
 
+targets:
+  release:
+    dependson:
+      - 'condition#release:and'
+    disableconditions: true
+    name: 'deps(github): bump Action release for actions/checkout from v4 to {{ source "release" }}'
+    kind: 'yaml'
+    sourceid: 'release'
+    transformers:
+      - addprefix: '"actions/checkout@'
+      - addsuffix: '"'
```

Comment:
I think it would be better to also include the version as a comment.
Something like:

```suggestion
      - addsuffix: '"'
      - addprefix: "# {{ source "release" }}"
```

The goal is to quickly identify the version associated to the hash and comment are allowed in github action files